README.md

```diff
@@ -0,0 +1,225 @@
+
+```bash
+# create the configmap for tripleo-deploy-config-custom
+oc create configmap -n openstack tripleo-deploy-config-custom --from-file=tripleo-deploy-config-custom/ --dry-run -o yaml | oc apply -f -
+
+# create the configmap for netconfig
+oc create configmap -n openstack tripleo-net-config --from-file=net-config/ --dry-run -o yaml | oc apply -f -
+```
+
+3) Create a [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) for your OpenStackControlPlane. This secret will provide the default password for your OSP Controller VMs.
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: userpassword
+  namespace: openstack
+data:
+  # 12345678
+  NodeRootPassword: MTIzNDU2Nzg=
+```
+
+If you write the above YAML into a file called ctlplane-secret.yaml you can create the Secret via this command:
+
+```bash
+oc create -f ctlplane-secret.yaml
+```
+
+3) Define your OpenStackControlPlane custom resource. The OpenStackControlPlane custom resource provides a cental place to create and scale VMs used for the OSP Controllers along with any additional vmsets for your deployment. At least 1 Controller VM is required for a basic demo installation and per OSP High Availability guidelines 3 Controller VMs are recommended.
+
+```yaml
+apiVersion: osp-director.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: overcloud
+  namespace: openstack
+spec:
+  openStackClientImageURL: quay.io/openstack-k8s-operators/tripleo-deploy:latest
+  passwordSecret: userpassword
+  virtualMachineRoles:
+    controller:
+      roleName: Controller
+      roleCount: 1
+      networks:
+        - ctlplane
+      cores: 6
+      memory: 12
+      diskSize: 40
+      baseImageURL: http://download.devel.redhat.com/brewroot/packages/rhel-guest-image/8.3/417/images/rhel-guest-image-8.3-417.x86_64.qcow2
+      baseImageVolumeName: openstack-base-img
+      storageClass: host-nfs-storageclass
+```
+
+If you write the above YAML into a file called openstackcontrolplane.yaml you can create the OpenStackControlPlane via this command:
+
+```bash
+oc create -f openstackcontrolplane.yaml
+```
+
+4) Define an OpenStackBaremetalSet to scale out OSP Compute hosts. The OpenStackBaremetal resource can be used to define and scale Compute resources and optionally be used to define and scale out baremetal hosts for other types of TripleO roles. The example below defines a single Compute host to be created:
+
+```yaml
+apiVersion: osp-director.openstack.org/v1beta1
+kind: OpenStackBaremetalSet
+metadata:
+  name: compute
+  namespace: openstack
+spec:
+  # How many nodes to provision
+  replicas: {{ osp_compute_count }}
+  # The image to install on the provisioned nodes
+  baseImageUrl: http://172.22.0.1/images/{{ osp_controller_base_image_url | basename }}
```

Comment:
this probably needs more explanation from where this image needs to be reachable?

Reply:
got it.